app/views/base/page.scala

```diff
@@ -110,10 +111,11 @@ object page:
           dataDev,
           dataVapid := (ctx.isAuth && env.security.lilaCookie.isRememberMe(ctx.req))
             .option(env.push.vapidPublicKey),
-          dataUser     := ctx.userId,
-          dataSoundSet := pref.currentSoundSet.toString,
-          dataSocketDomains,
-          pref.isUsingAltSocket.option(dataSocketAlts),
+          dataUser                         := ctx.userId,
+          dataSoundSet                     := pref.currentSoundSet.toString,
+          attr("data-socket-domains")      := SocketTest.socketEndpoints(netConfig).mkString(","),
+          attr("data-socket-test-user")    := SocketTest.isUserInTestBucket(netConfig),
```

Comment:
this hashes the username 2 times for 1 page loaded, so about 4000 hashes per second. I hope it's fast.